Extension/src/LanguageServer/client.ts

```diff
@@ -985,8 +1081,19 @@ export class DefaultClient implements Client {
                         // Listen for messages from the language server.
                         this.registerNotifications();
                     } else {
+                        // update all client configurations
                         this.configuration.CompilerDefaults = compilerDefaults;
```

Comment:
This line is also the same in both blocks and can be moved to after the else block.